assets/less/frontend-forms.less

```diff
@@ -3,8 +3,10 @@
 @mediaXS: 767px;
 @mediaSM: 991px;
 @mediaMD: 1200px;
-
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;1,300&display=swap');
```

Comment:
We should not import the font as it may conflict with the theme.

Reply:
@tareq1988 If you please guide here for alternative implementation.

Reply:
Just remove the font from here